easybuild/easyconfigs/m/Maude/Maude-3.1-GCCcore-10.2.0.eb

```
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'Maude'
+version = '3.1'
+
+homepage = "https://maude.cs.illinois.edu/w/index.php/The_Maude_System"
+description = """Maude is a high-performance reflective language and system supporting both equational and rewriting
+ logic specification and programming for a wide range of applications. Maude has been influenced in important ways by
+ the OBJ3 language, which can be regarded as an equational logic sublanguage. Besides supporting equational
+ specification and programming, Maude also supports rewriting logic computation. """
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://maude.cs.illinois.edu/w/images/d/d3/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['b112d7843f65217e3b5a9d40461698ef8dab7cbbe830af21216dfb924dc88a2f']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Bison', '3.7.1'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('BuDDy', '2.4'),
+    ('GMP', '6.2.0'),
+    ('libsigsegv', '2.13'),
+    ('libtecla', '1.6.3'),
+    ('Yices', '2.6.2'),
+]
+
+# Building in parallel sometimes errors
+maxparallel = 1
+
+runtest = 'check'
+
+modextravars = {'MAUDE_LIB': '%(installdir)s/share'}
+
+sanity_check_paths = {
+    'files': ['share/%s.maude' % x for x in ['file', 'linear', 'prelude', 'smt']],
```

@branfosj Also check for `bin/maude` here?

```suggestion
    'files': ['bin/maude'] + ['share/%s.maude' % x for x in ['file', 'linear', 'prelude', 'smt']],